{"data_version": "4.0","references": {"reference_data": [{"url": "https://github.com/rohe/pysaml2/issues/417"}]},"description": {"description_data": [{"lang": "eng","value": "Python package pysaml2 version 4.4.0 and earlier reuses the initialization vector across encryptions in the IDP server, resulting in weak encryption of data."}]},"data_type": "CVE","affects": {"vendor": {"vendor_data": [{"product": {"product_data": [{"version": {"version_data": [{"version_value": "4.4.0 and older"}]},"product_name": "pysaml2"}]},"vendor_name": "pysaml2"}]}},"CVE_data_meta": {"DATE_ASSIGNED": "2017-08-22T17:29:33.475589","ID": "CVE-2017-1000246","ASSIGNER": "kurt@seifried.org","REQUESTER": "predrag.gruevski@gmail.com"},"data_format": "MITRE","problemtype": {"problemtype_data": [{"description": [{"lang": "eng","value": "Initialization vector (IV) reuse"}]}]}}